packages/apps/human-app/frontend/src/modules/worker/components/jobs/available-jobs/components/mobile/available-jobs-table-jobs-list-mobile.tsx

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable camelcase -- ... */
+import { Grid, List, Paper, Stack, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useEffect, useState } from 'react';
+import { Button } from '@/shared/components/ui/button';
+import { useJobsFilterStore } from '@/modules/worker/hooks/use-jobs-filter-store';
+import { Alert } from '@/shared/components/ui/alert';
+import { getNetworkName } from '@/modules/smart-contracts/get-network-name';
+import { defaultErrorMessage } from '@/shared/helpers/default-error-message';
+import type { AvailableJob } from '@/modules/worker/services/available-jobs-data';
+import { useInfiniteGetAvailableJobsData } from '@/modules/worker/services/available-jobs-data';
+import { Loader } from '@/shared/components/ui/loader';
+import { EvmAddress } from '@/modules/worker/components/jobs/evm-address';
+import { Chip } from '@/shared/components/ui/chip';
+import { RewardAmount } from '@/modules/worker/components/jobs/reward-amount';
+import { ListItem } from '@/shared/components/ui/list-item';
+import { useColorMode } from '@/shared/hooks/use-color-mode';
+import { AvailableJobsAssignJobButtonMobile } from '@/modules/worker/components/jobs/available-jobs/components/mobile/available-jobs-assign-job-button-mobile';
+import { type JobType } from '@/modules/smart-contracts/EthKVStore/config';
+
+export function AvailableJobsTableJobsListMobile() {
+  const { t } = useTranslation();
+  const { colorPalette } = useColorMode();
+  const [allPages, setAllPages] = useState<AvailableJob[]>([]);
+  const { filterParams, setPageParams, resetFilterParams } =
+    useJobsFilterStore();
+  const {
+    data: tableData,
+    status: tableStatus,
+    isError: isTableError,
+    error: tableError,
+    fetchNextPage,
+    hasNextPage,
+  } = useInfiniteGetAvailableJobsData();
+
+  useEffect(() => {
+    if (!tableData) return;
+    const pagesFromRes = tableData.pages.flatMap((pages) => pages.results);
+
+    setAllPages((state) =>
+      filterParams.page === 0 ? pagesFromRes : [...state, ...pagesFromRes]
+    );
+  }, [tableData, filterParams.page]);
+
+  useEffect(() => {
+    return () => {
+      resetFilterParams();
+    };
+  }, [resetFilterParams]);
+
+  return (
+    <Stack flexDirection="column">
+      {isTableError && (
+        <Alert color="error" severity="error">
+          {defaultErrorMessage(tableError)}
+        </Alert>
+      )}
+      {tableStatus === 'pending' && (
+        <Stack alignItems="center" justifyContent="center">
+          <Loader size={90} />
+        </Stack>
+      )}
+
+      {allPages.map((d) => (
+        <Paper
+          key={crypto.randomUUID()}
```

Comment:
you can not use it here, because key must be deterministic between every render process...
please use sth like `${d.escrow_address}-${d.chain_id}`

https://dev.to/elyasshamal/generating-unique-ids-with-cryptorandomuuid-in-react-2d1e